frontend/src/components/storage/StorageDashboard.jsx

```diff
@@ -81,6 +81,24 @@ const StorageDashboard = () => {
     storageLocations: 0,
   });
 
+  // Callback for child components to refresh metrics (OGC-144: real-time updates per FR-057b, FR-057c)
```

Comment:
This is where the mapping gets tricky I think. I.e., OGC-144 makes sense, but since the spec itself is called 001-sample-storage, this comment becomes hard for a human to track down.

Reply:
great point - any FR references should also reference the feature itself. I think when we get through this push we can reset and clean up the specs dir a bit, and going forward have a cleaner approach